runtime/eoapi/raster/pyproject.toml

```diff
@@ -19,7 +19,7 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-    "titiler.pgstac==0.5.1",
+    "titiler.pgstac==0.7.0",
```

Comment:
@vincentsarago will it be difficult to upgrade to `titiler.pgstac==0.8.0`? I am eager to try out the new `/feature` endpoint from https://github.com/stac-utils/titiler-pgstac/pull/125

Reply:
nevermind, it's already done! :pray:

Reply:
😅 I just pushed an update https://github.com/developmentseed/eoAPI/commit/9383516e6e33637686ab9292a3530ef6d40c1e64

but I didn't enable the `/bbox` and `/feature` endpoint by default